Examples/Detectors/GenericDetector/src/BuildGenericDetector.cpp

```diff
@@ -11,10 +11,8 @@
 #include <cmath>
 #include <numbers>
 
-namespace ActsExamples::Generic {
-
 /// helper method for cylinder
-std::vector<Acts::Vector3> modulePositionsCylinder(
+std::vector<Acts::Vector3> ActsExamples::Generic::modulePositionsCylinder(
```

Comment:
This is the opposite of what you did for the majority of otherfiles.

Reply:
Right this is not super consistent. I personally prefer using `namespace ActsExamples {}` in source files but for binding to free functions this can be annoying since the compiler will not know that the declaration and the definition are actually the same. I usually then spell out the full namespace or at least the bottom level one. So I could also change it to `Generic::modulePositionsCylinder` to stay consistent with that approach.

What do you think?

Reply:
Ah, I didn't think of that actually. Let's keep it like this if it has an advantage.